wg-cloud-provider/draft-20171128-controller-manager.md

```diff
@@ -0,0 +1,294 @@
+---
+kep-number: draft-20171127
+title: Cloud Provider Controller Manager
+authors:
+  - "@cheftako"
+  - "@calebamiles"
+owning-sig: sig-apimachinery
+participating-sigs:
+  - sig-storage
+  - sig-apps
+  - sig-network
+reviewers:
+  - "@wlan0"
+  - "@calebamiles"
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2017-11-27
+last-updated: 2017-11-27
```

Comment:
meta note: Seems like a bad idea to keep these fields inside a doc in a source control system, they are tracked by git.

Reply:
I made exactly that same point @lavalamp...but was shouted down